<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employee_add_product_dialog</class>
 <widget class="QDialog" name="employee_add_product_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Manufacturer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Expiry date:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_le">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="manu_le">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>620</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Product information:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="price_spin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>1000000000</number>
   </property>
  </widget>
  <widget class="QComboBox" name="category_combo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Vegetable and Fruit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dairy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Beverage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Snack</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>None-Food</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="exp_date">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="amount_spin">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../images/images.qrc">:/addProduct.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../images/images.qrc"/>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
